.env

```diff
@@ -162,3 +162,5 @@ MAINNET_SEND_MANY_CONTRACT_ID=SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE.send-man
 # of openapi.yaml for test / development NODE_ENV. 
 # For production, /doc is not served if this env var is not provided.
 # API_DOCS_URL="https://docs.hiro.so/api" 
+
+# TSV_IMPORT_IN_PROCESS=false
```

Comment:
Since this ENV is for internal use only we shouldn't expose it here (maybe users can get confused and set it to true can cause some issues)